### PR TITLE
Adds tabby cat knowledge to InstructLab

### PR DESCRIPTION
**Pull request by jregehr**
If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Adds Tabby Cat knowlege**

Signed-off-by: Regehr, Jonathan <10343188+jregehr@users.noreply.github.com>

**Contribution checklist**

- [x] The contribution was tested with `ilab generate`
- [x] No errors or warnings were produced by `ilab generate`
- [x] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [x] The `qna.yaml` file contains at least 5 `seed_examples`
- [x] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [x] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [x] Content does not include PII or otherwise sensitive or confidential information
- [x] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

**Comment by jjasghar** [comment]
@instructlab-bot precheck

**Comment by jjasghar** [comment]
the `precheck` seems like the model already knows what a tabby cat is, @mingxzhao or @bjhargrave can you confirm what I'm seeing also?

**Comment by jregehr** [comment]
JJ, I felt like my first PR was really messed up so I closed the PR,
deleted my fork, made a new fork, re-added the files, and made a new PR.

Sorry for the confusion.

- Jonathan
[sent from my mobile]

File: knowledge/animals/tabby_cat/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+created_by: Jonathan Regehr
```
Comment (jjasghar) [inline knowledge/animals/tabby_cat/qna.yaml:1]: Please put your GitHub Username here.

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix the created_by then we can run the linter.

**Comment by jregehr** [comment]
Fixed!
- Jonathan

**Comment by jjasghar** [comment]
Please fix the DCO issue.

**Comment by jregehr** [comment]
@jjasghar We talked about this PR at DevOpsDays KC :)

**Comment by jjasghar** [comment]
Here's another linter broken @bjhargrave, it's not pulling the newest script for some reason. I'm not sure what's going on here.

**Comment by bjhargrave** [comment]
> Here's another linter broken @bjhargrave, it's not pulling the newest script for some reason. I'm not sure what's going on here.

It needed a rebase to get the #1090 fix.

**Comment by jjasghar** [comment]
@instructlab-bot precheck

**Comment by jjasghar** [comment]
@nerdalert or @vishnoianil can you take a look at this one, the Precheck bot is :sadpanda:

**Comment by nerdalert** [comment]
@instructlab-bot precheck

**Comment by nerdalert** [comment]
@jjasghar Its a known bug, will bump it up. We have seen a couple of times that git manages to get into a weird state. Tossing the fork on the worker resolves it. Thanks for the heads up!

**Comment by ljmwaugh** [comment]
I've merged your qna.yaml with another tabby_cat qna.yaml and the new PR is here
https://github.com/instructlab/taxonomy/pull/1118

**Comment by jjasghar** [comment]
Closing per https://github.com/instructlab/taxonomy/pull/1118.

**Pull request closed without merging**